src/app/core/_components/forms/simple-forms/form.component.html

```diff
@@ -1,6 +1,6 @@
 <app-dynamic-form
                 *ngIf="isloaded"
-                [subtitle]='title'
+                [title]='title'
```

Comment:
Use double quotes instead of single quotes like the other strings in this form

Reply:
Changed from single quotes to double quotes